### ISSUE TITLE
TypeError: Cannot read property '_id' of null

### ISSUE THREAD

**Issue by mkahvi** [espositos/fvtt-tokenactionhud#152]
I've been getting following error a lot lately:
```
foundry.js:2499 TypeError: Cannot read property '_id' of null
    at TokenActionHUD.validTokenChange (tokenactionhud.js:375)
    at init.js:40
    at Function._call (foundry.js:2496)
    at Function.callAll (foundry.js:2456)
    at Function._handleUpdateEmbeddedEntity (foundry.js:30047)
    at Function._handleUpdateEmbeddedEntity (foundry.js:34301)
    at n.<anonymous> (foundry.js:29410)
    at n.emit (index.js:83)
    at n.onevent (index.js:83)
    at n.onpacket (index.js:83)
```
I don't know what triggers it, since I notice it usually only after coming back to a server after long time being AFK / not playing (long idle login hidden in some background tab).

~~I'm guessing updateToken event sometimes gives null as token value for some reason?~~
NVM, that was wrong: It's because the player doesn't have character configured.  
> token.actorId === game.user.character._id;

Player is controlling multiple characters, so they don't have a character selected in player config.

FVTT 0.7.9, PF1e 0.76.13, TAHUD 0.10.23

**Comment by erithtotl** [comment]
Simple fix:

            return this.isRelevantToken(token) || token.actorId === game?.user?.character?._id;

**Comment by espositos** [comment]
PR is merged into the last patch.
